components/feature/loginsexceptions/src/androidTest/java/mozilla/components/feature/loginsexceptions/LoginExceptionStorageTest.kt

```diff
@@ -0,0 +1,140 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.loginsexceptions
+
+import android.content.Context
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.paging.PagedList
+import androidx.room.Room
+import androidx.room.testing.MigrationTestHelper
+import androidx.sqlite.db.framework.FrameworkSQLiteOpenHelperFactory
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.platform.app.InstrumentationRegistry
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.runBlocking
+import mozilla.components.feature.loginsexceptions.db.LoginExceptionDatabase
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertNull
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+
+@Suppress("LargeClass")
+class LoginExceptionStorageTest {
```

Comment:
Lets make sure this component is listed here so that on-device tests for it run on Firebase:
https://github.com/mozilla-mobile/android-components/blob/master/taskcluster/ci/test/kind.yml#L51

I do not remember whether this is the only thing needed, @Amejia481 recently did this for an other component and may know. :)

Reply:
Yes, that's correct. You only need to add another entry.

Reply:
@ekager to verify that android tests were added correctly just use `bors try` and you will see a task with the symbol name that you chosen with the prefix `test-` e.g `test-YOUR_SYMBOL_NAME`

Reply:
💃 
<img width="449" alt="Screen Shot 2020-07-06 at 3 54 53 PM" src="https://user-images.githubusercontent.com/25442310/86635443-14555d00-bfa1-11ea-9050-1e25a53af9da.png">